flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/ConnectedStreams.scala

```diff
@@ -101,30 +101,31 @@ class ConnectedStreams[IN1, IN2](javaStream: JavaCStream[IN1, IN2]) {
   }
 
   /**
-   * Applies the given [[TimelyCoFlatMapFunction]] on the connected input streams,
+   * Applies the given [[CoProcessFunction]] on the connected input streams,
    * thereby creating a transformed output stream.
    *
    * The function will be called for every element in the streams and can produce
    * zero or more output. The function can also query the time and set timers. When
    * reacting to the firing of set timers the function can emit yet more elements.
    *
```

Comment:
The function will be called for every element in the input streams and can produce zero or more
   * output elements. Contrary to the [[ConnectedStreams#flatMap]] function, this function can also
   * query the time and set timers. When reacting to the firing of set timers the function can directly
   * emit elements and/or register yet more timers.